serverless-cljs-plugin/serverless_lumo/build.cljs

```diff
@@ -22,7 +22,8 @@
     - https://archiverjs.com/docs/Archiver.html#file"
   [output-path zip-opts compiler-opts]
 
-  (let [archiver      (archiver "zip" #js {:zlib {:level 9}})
+  (let [archiver      (archiver "zip" #js {:zlib {:level 9}
+                                           :comment (str "Generated by serverless-cljs-plugin on" (.toISOString (js/Date.)))})
```

Comment:
This is mirroring the one in `applyZipExclude` which is not always called, so here we take care of that.

Reply:
It's a very minor issue, but perhaps `const comment = ...` before the first use might be easier to follow than having very slightly different timestamps for each entry.

Reply:
Ok will do